### ISSUE TITLE
Get "Demonstrate_EndToEnd_Orbit_Precalc.ipynb" Working

### ISSUE THREAD

**Issue by matthewjohnpayne** [matthewjohnpayne/cheby_checker#8]
@Mikea1985 to step through the 3 cells in "Demonstrate_EndToEnd_Orbit_Precalc.ipynb" 
Establish whether each cell works 
If not, dig into the underlying code to understand / diagnose the problem.
Then either fix the problems directly, or create a list of questions / suggestions for @matthewjohnpayne to work on.

**Comment by matthewjohnpayne** [comment]
@Mikea1985 did you make changes to mpc_nbody that were not added/pushed? 
 - When I run the tests in "test_run_nbody.py" they mostly fail
 - I don't see any recent changed to mpc_nbody.py (I guess you probably had to make changes to get the code working). 
 - Also, the notebook Demonstrate_EndToEnd_Orbit_Precalc.ipynb doesn't work

**Comment by Mikea1985** [comment]
I have only made a few exploratory edits to test_run_nbody.py while I was trying to identify why three tests were failing; once I realised that they were failing due to Reboundx/ephem_forces accuracy bug, I abandoned all changes I had made. I have thus made no changes (worth keeping) to either test_run_nbody.py or mpc_nbody.py since early August. Are you getting more than the 3 failures and 2 x-fails (eXpected failures), of the 15 total tests? 

Indeed, Demonstrate_EndToEnd_Orbit_Precalc.ipynb does not work; I did not manage to finish getting it working on Friday. It's just the last cell that doesn't work, correct?

**Comment by matthewjohnpayne** [comment]
No, Demonstrate_EndToEnd_Orbit_Precalc.ipynb fails on the first cell, with a message about needing to supply "tstart" and then when I fix that, a further message about needing to supply 'vectors' and 'epoch'. 
 - I can go through and fix these, but you had to do this yourself previously, so something has got out of sync. 
 - It could easily be that I fucked-up merging your commit, or it could just be that some part of your changes didn't get added/committed.

I don't want you to have to spend time going through this, so instead it would be helpful to me if you could email me copies of the following codes so that I can do a side-by-side comparison to see what the differences are:
 - mpc_nbody.py
 - parse_input.py
 - Demonstrate_EndToEnd_Orbit_Precalc.ipynb
